src/Microsoft.Diagnostics.ExtensionCommands/FindPointersInCommand.cs

```diff
@@ -0,0 +1,517 @@
+﻿using Microsoft.Diagnostics.Runtime;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using static Microsoft.Diagnostics.ExtensionCommands.MAddressCommand;
+using Microsoft.Diagnostics.DebugServices;
+using System.IO;
+using System.Diagnostics;
+
+namespace Microsoft.Diagnostics.ExtensionCommands
+{
+    [Command(Name = "findpointersin", Help="Finds pointers to the GC heap within the given memory regions.")]
+    public class FindPointersInCommand : ExtensionCommandBase
```

Comment:
I recommend using CommandBase directly instead of ExtensionCommandBase because you don't need or use the CLRMDHelper instance it provides.  It would mean that you would have to rename ExtensionInvoke to Invoke and add the HelpInvoke attribute to your GetDetailedHelp functions (and adding a WriteLine around the string). I have some upcoming improvements to those detailed help functions and doing it that way would make it easier.